Mixpanel/Mixpanel.m

```diff
@@ -696,6 +722,7 @@ - (void)archiveProperties
     [p setValue:self.people.unidentifiedQueue forKey:@"peopleUnidentifiedQueue"];
     [p setValue:self.shownNotifications forKey:@"shownNotifications"];
     [p setValue:self.timedEvents forKey:@"timedEvents"];
+    [p setValue:self.automaticEventsEnabled forKey:@"automaticEvents"];
```

Comment:
the reason I put this is that the next time the person opens the app, we already know he wants pretrack on so we wont have to "store and wait" for a decide response. If the user turns it off, the next time we get a decide response it would be sent to off. I like it this way because it is more graceful and Im not holding automatic events and not sending them for most cases (the normal case is its not toggled, the unlikelier case is it is toggled)